<launch>
   

    <group ns="typhoon_h480_0">

        <param name="drone_name" value="typhoon_h480"/>
        <param name="drone_id" value="0"/> 
        <param name="max_err" value="2.0" />     
    
        <node pkg="robocup" name="logic" type="logic" output="screen" >  
            <remap from="move_base/goal" to="drone_0_ego_planner_node/move_base_simple/goal" />
        </node>
    </group>

        <group ns="typhoon_h480_1">

        <param name="drone_name" value="typhoon_h480"/>
        <param name="drone_id" value="1"/> 
        <param name="max_err" value="2.0" />     
    
        <node pkg="robocup" name="logic" type="logic" output="screen" >  
            <remap from="move_base/goal" to="drone_1_ego_planner_node/move_base_simple/goal" />
        </node>
    </group>


        <group ns="typhoon_h480_2">

        <param name="drone_name" value="typhoon_h480"/>
        <param name="drone_id" value="2"/> 
        <param name="max_err" value="2.0" />     
    
        <node pkg="robocup" name="logic" type="logic" output="screen" >  
            <remap from="move_base/goal" to="drone_2_ego_planner_node/move_base_simple/goal" />
        </node>
    </group>

    <group ns="typhoon_h480_3">

        <param name="drone_name" value="typhoon_h480"/>
        <param name="drone_id" value="3"/> 
        <param name="max_err" value="2.0" />     
    
        <node pkg="robocup" name="logic" type="logic" output="screen" >  
            <remap from="move_base/goal" to="drone_3_ego_planner_node/move_base_simple/goal" />
        </node>
    </group>

   <group ns="typhoon_h480_4">

        <param name="drone_name" value="typhoon_h480"/>
        <param name="drone_id" value="4"/> 
        <param name="max_err" value="2.0" />     
    
        <node pkg="robocup" name="logic" type="logic" output="screen" >  
            <remap from="move_base/goal" to="drone_4_ego_planner_node/move_base_simple/goal" />
        </node>
    </group>

   <group ns="typhoon_h480_5">

        <param name="drone_name" value="typhoon_h480"/>
        <param name="drone_id" value="5"/> 
        <param name="max_err" value="2.0" />     
    
        <node pkg="robocup" name="logic" type="logic" output="screen" >  
            <remap from="move_base/goal" to="drone_5_ego_planner_node/move_base_simple/goal" />
        </node>
    </group>




</launch>